clang/lib/Headers/aiev2_vadd.h

```diff
@@ -1291,6 +1291,77 @@ INTRINSIC(v16int32) bfloat16_to_int(v16bfloat16 a, int shft) {
   return __builtin_aiev2_v16bf16_to_v16i32(a, shft);
 }
 
+INTRINSIC(unsigned int) lt(v16float v1, v16float v2) {
+  v32bfloat16 a = broadcast_zero_to_v32bfloat16();
+  v32bfloat16 b = broadcast_zero_to_v32bfloat16();
+  a = insert(a, 0, to_v16bfloat16(sub((v16accfloat)v1, (v16accfloat)v2)));
+  return lt(a, b);
```

Comment:
That must generate lots of instructions, right?
If there a reason we don't do `return lt(to_v16bfloat16(v1), to_v16bfloat16(v2)`? Is that because of precision?

Reply:
Yes this is due to precision. If you look at the test, I think the total number of instructions isn't too bad either